### PR TITLE
Formbot: small updates on training data

### PR DESCRIPTION
**Pull request by JustinaPetr**
**Proposed changes**:
I want to reuse this example for the forms tutorial. Without changing too much on the code side, proposing a few improvements on the NLU side and additional intent to make the conversation a bit more natural.

- A few more examples to NLU training data and added greet intent
- Updated stories and domain accordingly
- Added a short readme

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: examples/formbot/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+can build a simple restaurant search assistant capable of recommending
+restaurants based on user preferences.
+
+## What’s inside this example?
+
+This example contains some training data and the main files needed to build an 
+assistant on your local machine. The formbot consists of the following files:
+
+- **data/nlu_data.md** contains training examples for NLU model  
+- **data/nlu_data.md** contains training stories for Core model
```
Comment (akelad) [inline examples/formbot/README.md]:
```suggestion
- **data/stories.md** contains training stories for Core model  
```

File: examples/formbot/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Formbot - an example which demonstrates the implementation of FormAction
+
+Formbot example is designed to help you understand how FormAction works and how
```
Comment (akelad) [inline examples/formbot/README.md]:
```suggestion
Formbot example is designed to help you understand how the FormAction works and how
```

File: examples/formbot/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+restaurants based on user preferences.
+
+## What’s inside this example?
+
+This example contains some training data and the main files needed to build an 
+assistant on your local machine. The formbot consists of the following files:
+
+- **data/nlu_data.md** contains training examples for NLU model  
+- **data/nlu_data.md** contains training stories for Core model  
+- **actions.py** contains the implementation of custom FormAction
```
Comment (akelad) [inline examples/formbot/README.md]:
```suggestion
- **actions.py** contains the implementation of a custom FormAction  
```

File: examples/formbot/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## What’s inside this example?
+
+This example contains some training data and the main files needed to build an 
+assistant on your local machine. The formbot consists of the following files:
+
+- **data/nlu_data.md** contains training examples for NLU model  
+- **data/nlu_data.md** contains training stories for Core model  
+- **actions.py** contains the implementation of custom FormAction  
+- **domain.yml** contains the domain of the assistant  
+- **endpoints.yml** contains webhook configuration for the custom action
```
Comment (akelad) [inline examples/formbot/README.md]:
```suggestion
- **endpoints.yml** contains the webhook configuration for the custom action  
```

File: examples/formbot/data/stories.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
 ## happy path
+* greet
+	- utter_greet
```
Comment (akelad) [inline examples/formbot/data/stories.md]: can we make sure the indents for the `utter_greet` align with the rest of the actions?

Comment (JustinaPetr) [reply examples/formbot/data/stories.md]: Oh, good catch. Fixed now.

**Pull request merged**